From f0ea5bdd83f1cc7de3db793f33755b9196b42f26 Mon Sep 17 00:00:00 2001
From: Alexandru Costache <alexandru@balena.io>
Date: Wed, 14 Apr 2021 14:53:15 +0200
Subject: [PATCH] nitrogen8mm-dwe-1g: Use BalenaOS configs

Integrate the 1G RAM version with BalenaOS.

We also dropped cmd_custom env var that set ram
from cmdline because that is not present in the new
uboot that came with the board and should
of course not be necessary anymore.

Upstream-status: Inappropriate [configuration]
Signed-off-by: Alexandru Costache <alexandru@balena.io>
---
 configs/dwe_1gch1_defconfig | 9 ++++++++-
 1 file changed, 8 insertions(+), 1 deletion(-)

diff --git a/configs/dwe_1gch1_defconfig b/configs/dwe_1gch1_defconfig
index c3b91c016f..3775ab2301 100644
--- a/configs/dwe_1gch1_defconfig
+++ b/configs/dwe_1gch1_defconfig
@@ -8,7 +8,7 @@ CONFIG_SPL_LIBCOMMON_SUPPORT=y
 CONFIG_SPL_LIBGENERIC_SUPPORT=y
 CONFIG_SYS_MALLOC_F_LEN=0x10000
 CONFIG_NR_DRAM_BANKS=1
-CONFIG_ENV_SIZE=0x2000
+CONFIG_ENV_SIZE=0x3000
 CONFIG_ENV_OFFSET=0xffffe000
 CONFIG_DDR_RANK_BITS=0
 CONFIG_DDR_CHANNEL_CNT=1
@@ -134,3 +134,10 @@ CONFIG_USBNET_DEVADDR="00:19:b8:00:00:02"
 CONFIG_USBNET_HOST_ADDR="00:19:b8:00:00:01"
 CONFIG_USB_HOST_ETHER=y
 CONFIG_IMX_WATCHDOG=y
+CONFIG_FAT_WRITE=y
+CONFIG_ENV_IS_NOWHERE=y
+CONFIG_CMD_SAVEENV=n
+CONFIG_FAT_WRITE=y
+CONFIG_CMD_EXPORTENV=y
+CONFIG_CMD_IMPORTENV=y
+CONFIG_CMD_FS_UUID=y
-- 
2.17.1

